### PR TITLE
Make -[IGListAdapter updater] Public, Read-Only

### PR DESCRIPTION
**Pull request by Adlai-Holler**
Hey IGListKit folks! Would you be willing to expose this, so that other objects that are given a list adapter can inspect its updater configuration?

## Pull request checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [x] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by jessesquires** [comment]
I'm fine with this change. What's your use case?

> /cc: @rnystrom ?

**Comment by Adlai-Holler** [comment]
I'd like to be able to assert that a list adapter passed into an API does not use `IGListReloadDataUpdater` and that the updater is configured with `allowsBackgroundReloading=NO`. I could modify the API to require the user to keep track of the updater and pass it in alongside the list adapter, but it seems like unnecessary hassle.

**Comment by jessesquires** [comment]
Hmm... but you have to initialize the adapter with an `id<IGListUpdatingDelegate>` couldn't you enforce this then?

**Comment by Adlai-Holler** [comment]
Since the component that receives the adapter is not the one who initializes it, the adapter's owner would have to keep track of the updater separately and provide them both into this API. So `@property ASCollectionNode.listAdapter` would have to become `-[ASCollectionNode setListAdapter:updater:]`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ This release closes the [2.1.0 milestone](https://github.com/Instagram/IGListKit
 
 - Added CocoaPods subspec for diffing, `IGListKit/Diffing` and an [installation guide](https://instagram.github.io/IGListKit/installation.html). [Sherlouk](https://github.com/Sherlouk) [(#368)](https://github.com/Instagram/IGListKit/pull/368)
 
+- `IGListAdapter.updatingDelegate` is now public (read-only). [Adlai-Holler](https://github.com/Adlai-Holler) [(#379)](https://github.com/Instagram/IGListKit/pull/379)
```
Comment (rnystrom) [inline CHANGELOG.md]: Super nit: all of our changelog is in ObjC. Can we change the first part to `-[IGListAdapter updatingDelegate]`

**Comment by jessesquires** [comment]
@rnystrom - want to just import this and forward-fix internally?

**Comment by Adlai-Holler** [comment]
@rnystrom The internal (readwrite) version of the property unfortunately needs to stay, because the tests use it:

screen shot 2017-01-03 at 3 07 40 pm

**Comment by Adlai-Holler** [comment]
Cool. I've renamed `updatingDelegate -> updater` and pushed ✨

**Pull request closed without merging**